.appveyor.yml

```diff
@@ -0,0 +1,17 @@
+environment:
+  matrix:
+    - TOXENV: py36
+    - TOXENV: py35
+    - TOXENV: py34
+    - TOXENV: py33
+    - TOXENV: py27
+    - TOXENV: py26
+
```

Comment:
I don't suppose anyone has written a command for Appveyor that will import the relevant bits from .travis.yml to avoid duplication?

Reply:
Yep, sigh. At least it's overall a pretty short file now.

I think the ideal thing would be if appveyor and travis supported tox, then you'd only have the tox file. One problem on travis is that the 'build stages' I use here were added to travis fairly recently (they are still beta), so not even all of travis's own stuff seems to work with them yet. Otherwise I could maybe have used something like https://tox-travis.readthedocs.io/en/stable/.

I have used tox before on another project in the past (so it's been around a bit), and liked it. What I mainly like is that: 
 * the config is short...
 * ...yet it covers a lot (particularly the testing of packages)
 * and the config is separate from your code, i.e. does not tie itself in to your code. So (a) it is not a big deal to replace if something else comes along, and (b) I can use tox locally, but e.g. you don't have to if you don't want (you can just run one or more of the commands from tox.ini manually if you want, like `nosetests`).
 
It would be great if there were something providing similar functionality in the conda world, and if there were a conda plugin for travis, etc :)